examples/discovery-search-app/cypress/integration/passages/spec.ts

```diff
@@ -99,5 +99,50 @@ describe('Passage Results', () => {
       cy.get('@combinedResult').contains('View passage in document');
       cy.get('@combinedResult').contains('View table in document');
     });
+
+    describe('and clicking on "CI Document" for a result', () => {
+      beforeEach(() => {
+        cy.get('button[data-testid="search-result-element-preview-button"]')
+          .contains('View table in document')
+          .click();
+        cy.get('ul')
+          .contains('CI')
+          .should('exist');
+        cy.get('ul')
+          .contains('CI')
+          .click();
+      });
```

Comment:
Doing gets on specific tags is fragile: if the underlying implementation changes from `ul` to `div`, for example, while keeping user-facing stuff basically the same, then this test will fail.

If there is no unique text to latch on to, then this looks like a good place to add a `data-testid`.